addons/event_sale/views/sale_order_views.xml

```diff
@@ -5,12 +5,14 @@
         <field name="model">sale.order</field>
```

Comment:
> Commit message
"before" in the commit message seems weird. The button just moved, right? Not removed.
Also I usually expect "before" to just explain the previous behavior not the reason for the change.

Maybe just say something along the lines of "the attendee count was moved with the preview button. We make it a stat button as it makes more sense for the purpose it serves, and will prevent further unintended re-locations of the button"

Also task id should be in the format `task-[id]`, notably this prevents the id from being interpreted as an Issue/PR ID or a commit hash.

Reply:
Hello @thiryr 

Changes are done please have a look.

Thanks !!